exporter_test.go

```diff
@@ -134,8 +134,8 @@ func testExporter(t *testing.T, when spec.G, it spec.S) {
 
 		exporter = &lifecycle.Exporter{
 			Buildpacks: []lifecycle.GroupBuildpack{
-				{ID: "buildpack.id", Version: "1.2.3"},
-				{ID: "other.buildpack.id", Version: "4.5.6", Optional: false},
+				{ID: "buildpack.id", Version: "1.2.3", API: "0.1"},
+				{ID: "other.buildpack.id", Version: "4.5.6", API: "0.1", Optional: false},
```

Comment:
I still feel like we should set these values here: https://github.com/buildpacks/lifecycle/blob/d9c7de5395c1ba66682b3e548f8955a11b265c17/exporter_test.go#L137-L138, so we won't have to remember to set the api for every test case.